apps/dashboard/src/components/selects/CustomChainRenderer.tsx

```diff
@@ -26,9 +32,13 @@ export const CustomChainRenderer = ({
   const setIsOpenNetworkConfigModal = useSetIsNetworkConfigModalOpen();
   const addRecentlyUsedChain = useAddRecentlyUsedChainId();
   const setEditChain = useSetEditChain();
+  const chainQuery = useQuery({
+    queryKey: ["getChain", chain.id],
+    queryFn: () => getChain(`${chain.id}`),
+  });
```

Comment:
can't to this it will make way too many requests

Reply:
V5 only renders chains that are visible

Reply:
oh yeah, I can use use the chain object from useSupportedChains()